pyrdp/player/ReplayThread.py

```diff
@@ -44,41 +44,41 @@ def __init__(self, replay: Replay):
         self.lastSeekTime = 0
         self.requestedSpeed = 1
         self.replay = replay
+        self.timer = Timer()
```

Comment:
Were these changes really necessary?

Reply:
It was in my previous implementation. I forgot to completely revert this change after I changed the implementation, so no they were not necessary.